### ISSUE TITLE
Editable cell?

### ISSUE THREAD

**Issue by amueller** [jupyterlite/jupyterlite-sphinx#63]
This might just be a misunderstanding on my part, but I had expected `:replite:` provide an editable cell, instead it acts more like a repl, which means copy & pasting the code. That makes sense in that it prevents the user from messing with the example, but it doesn't feel very notebook-y to me and it was a bit confusing.

Did I understand the intention correctly, and is there a plan to have embedded executable cells as well?

**Comment by amueller** [comment]
I guess this relates to https://github.com/jupyterlite/jupyterlite/issues/148

**Comment by jtpio** [comment]
> I guess this relates to [jupyterlite/jupyterlite#148](https://github.com/jupyterlite/jupyterlite/issues/148)

Yes having some kind of editable cell like the mockups and examples in https://github.com/jupyterlite/jupyterlite/issues/148 are still on the radar.
